app/content/spatial-extension-bq/sql-reference/transformations.md

```diff
@@ -115,11 +115,11 @@ transformations.ST_CONCAVEHULL(geog, maxEdge, units)
 
 **Description**
 
-Takes a set of points and returns a concave hull Polygon or MultiPolygon. Internally, this uses turf-tin to generate geometries. https://turfjs.org/docs/#concave
+Takes a set of points and returns a concave hull Polygon or MultiPolygon. Internally, this uses turf-tin to generate geometries.
```

Comment:
Turf-tin reference

Reply:
Yes, I was wondering what to do with that one. To me it looks ok, as we are just explaining how it works internally, and we are avoiding links

Reply:
I'd remove it. For me it's an implementation detail that we don't need to expose.